python/nav/auditlog/__init__.py

```diff
@@ -17,8 +17,5 @@
 from __future__ import unicode_literals
 
 
-default_app_config = 'nav.auditlog.apps.AuditlogConfig'
```

Comment:
This was new in Django 3.2 - so I suppose this will actually make NAV non-compatible with previous versions of Django. Or am I misunderstanding?

Reply:
`default_app_config` was introduced in 1.7, works in 3.2 but leads to a warning, will not work in 4.0. So removing such lines means we must run on 3.2 or newer.

If we also edit INSTALLED_APPS to contain the full path to the app-config (in this specific case: "nav.auditlog.apps.AuditlogConfig", it'll still run on 2.2. It's a matter of preference, and whatever we choose we should probably document it in a code style doc. We *very* rarely add new apps to NAV though.. The latter style exists in case an app has alternative app configs, to be able to explicitly choose one of them. NAV doesn't have such and I think it is unlikely that it ever will.

In a different project `default_app_config` actually lead to not using the app-configs I thought I was using. If there was a typo in the value, a magically generated app config would be used instead, based on the location of `models.py`.